Vermine 2047/sheet.json

```
@@ -0,0 +1,8 @@
+{
+	"html": "layout_vermine.html",
+	"css": "style_vermine.css",
+	"authors": "BlackyB",
+	"roll20userid": "888572",
+	"preview": "",
+	"instructions": ""
```

```suggestion
	"instructions": "Vermine 2047 is a reboot of a French RPG called 'Vermine'."
```
Might as well have something in the description.